README.rst

```diff
@@ -153,10 +155,19 @@ You can restrict pairs by setting filtering function to ``filter_func`` at
          8: ['Brand Y', 'NT', 'Internal', 'Part-Time', 30]
          9: ['Brand X', '2000', 'Modem', 'Hourly', 10]
         10: ['Brand Y', 'XP', 'Modem', 'Contr.', 30]
-        11: ['Brand Y', '2000', 'Modem', 'Salaried', 15]
-        12: ['Brand Y', 'NT', 'Modem', 'Salaried', 10]
-        13: ['Brand Y', 'XP', 'Modem', 'Part-Time', 6]
-        14: ['Brand Y', '2000', 'Modem', 'Contr.', 60]
+        11: ['Brand X', '98', 'Internal', 'Contr.', 60]
+        12: ['Brand X', '2000', 'Internal', 'Salaried', 15]
+        13: ['Brand Y', 'NT', 'Modem', 'Salaried', 10]
+        14: ['Brand Y', 'XP', 'Modem', 'Part-Time', 6]
+        15: ['Brand X', '98', 'Internal', 'Hourly', 30]
+        16: ['Brand X', '2000', 'Internal', 'Contr.', 60]
+        17: ['Brand Y', 'NT', 'Modem', 'Hourly', 15]
+        18: ['Brand X', '98', 'Internal', 'Part-Time', 10]
+        19: ['Brand Y', 'XP', 'Modem', 'Salaried', 10]
+        20: ['Brand X', '2000', 'Internal', 'Salaried', 6]
+        21: ['Brand Y', 'NT', 'Modem', 'Part-Time', 60]
+        22: ['Brand X', '98', 'Internal', 'Hourly', 60]
+        23: ['Brand Y', 'XP', 'Modem', 'Salaried', 30]
```

Comment:
Taking into account filtering function, this output were lacking the following pairs:
```
98 - Hourly
98 - Contr.
98 - 10
98 - 30
98 - 60
NT - 15
2000 - 6
XP - 10
Salaried - 30
Hourly - 30
Hourly - 60
Part-Time - 60
```

Now they are all present in the resulting set.